pages/mesosphere/dcos/services/hive-metastore/1.0.0-3.0.0/getting-started/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Getting Started
+excerpt: Getting started with Hive Metastore
+title: Getting Started
+menuWeight: 20
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+---
+
+Getting started with a test instance of the DC/OS {{ model.techName }} service is straightforward.
+
+## Prerequisites
+
+- Depending on your security mode in Enterprise DC/OS, you may need to [provision a service account](/mesosphere/dcos/services/{{ model.serviceName }}/2.7.0-3.11.4/security/#provisioning-a-service-account) before installing. You will be able to create the service account only if you have a `superuser` permission.
```

Comment:
oops, this is also a wrong address pointing to Cassandra @SarahStegall

Reply:
Already fixed. :)